content/blog/2022/03/camel316-whatsnew/index.md

```diff
@@ -0,0 +1,108 @@
+Now `camel-jbang` supports reloading `.properties` files in reload mode.
+
+We also added support for loading `.java` source containing custom POJOs, processors,
+or type-converters which can be reloaded. This allows users to modify the source code,
+and let `camel-jbang` hot re-load changes.
+
+The `camel-jbang` now also supports Camel K _modeline_ configurations, and other parts
+from Camel K & Kamelets, that makes camel-jbang more on-par with Camel K and makes it possible to
+run many more Camel K files out-of-the-box.
+
+And we also made `camel-jbang` auto download DSLs, so for example It's possible to run a kotlin
+source file and automatic download the needed `camel-kotlin-dsl` JAR.
+
+A preliminary support for using dependency injection annotations from Spring, Quarkus, and CDI
+has been added, which bridges the gap between camel-jbang and using Camel K, Camel Spring Boot,
+or Camel Quarkus. The intention is to allow getting started with building Camel integrations
+using `camel-jbang` / [Camel Karavan](https://github.com/apache/camel-karavan) and then
+later transition to a regular Camel on Spring Boot or Quarkus project.
+We will continue working on this story for upcoming releases.
+
+See more details at the [Camel JBang documentation](/manual/camel-jbang.html)
+
+### Camel UI Designer
+
+The [Camel Karavan](https://github.com/apache/camel-karavan) project is progressing nicely, and during the development we have identified
+a number of _mistakes_ in the route model in Camel that has been improved and corrected.
+
+This overall makes it easier to build custom Camel tooling as there are less _model hacks_ you may need to
+implement in the tool.
+
+The changes to the model may affect users when upgrading, so make sure to read the [upgrade guide](/manual/camel-3x-upgrade-guide-3_16.html).
+
+### Camel Health Checks
+
+The `camel-health` has been overhauled to be made simpler to implement custom health-checks.
+
+### Camel Route Templates
+
+The XML and YAML DSL now also supports creating routes from route templates.
+TODO: Add code example
+https://issues.apache.org/jira/browse/CAMEL-17611
+
+### Component Headers
+
+TODO: Work started on marking up all headers in every components for automatic documentation, and tooling support.
+
+### Camel Kafka
+
+TODO: 
+
+We added specific health-checks for kafka producer and consumers that checks the kafka-client
+internals if the connectivity with the kafka brokers is healthy or not.
+
+We did other bug fixes, and made camel-kafka more roboust.
+
+### Quarkus
+
+The upcoming Camel Quarkus 2.8 release will be upgraded to Camel 3.16. 
+
+### Spring Boot
+
+We have upgraded to the latest Spring Boot 2.6 release.
+
+Added more unit tests to various starter JARs to increase the QA of this project.
+
+## Upgrading
+
+Make sure to read the [upgrade guide](/manual/camel-3x-upgrade-guide-3_16.html) if you are upgrading from a previous Camel version.
+
+## Release Notes
```

Comment:
Nice one @davsclaus. I'd also include the deprecation notice for the `camel-testcontainers*` components and a link to the blog post: https://camel.apache.org/blog/2022/02/camel-test-infra/

Probably something along the lines of: 
```
## Deprecated Components: Camel Testcontainers

In this release we are replacing the camel-testcontainers components with the camel-test-infra. These new set of test-related components simplify integration testing with Camel, reduce our maintenance effort, allow us to test edge case scenarios more easily and more. Please check the [blog post](https://camel.apache.org/blog/2022/02/camel-test-infra/) where we explained about the changes.

```